src/hyperion/parameters/gridscan.py

```diff
@@ -249,23 +173,21 @@ def panda_FGS_params(self) -> PandAGridScanParams:
 
     @property
     def grid_1_spec(self):
-        x_end = self.x_start_um + self.x_step_size_um * self.x_steps
-        y1_end = self.y_start_um + self.y_step_size_um * self.y_steps
+        x_end = self.x_start_um + self.x_step_size_um * (self.x_steps - 1)
+        y1_end = self.y_start_um + self.y_step_size_um * (self.y_steps - 1)
         grid_1_x = Line("sam_x", self.x_start_um, x_end, self.x_steps)
         grid_1_y = Line("sam_y", self.y_start_um, y1_end, self.y_steps)
-        grid_1_omega = Static("omega", self.grid1_omega_deg)
         grid_1_z = Static("sam_z", self.z_start_um)
-        return grid_1_x.zip(grid_1_z).zip(grid_1_omega) * ~grid_1_y
+        return grid_1_y.zip(grid_1_z) * ~grid_1_x
```

Comment:
I don't really understand why this has changed - I'm assuming there's a reason but I don't understand scanspecs enough to work out what this does

Reply:
because I got it wrong the first time, haha. it changes which axis is the slow/fast dimension. it wasn't a problem before because the scanspec was recalculated in the old params.

Reply:
Could: If you change this back e.g. remove the `-1` it's only picked up in a test for writing nexus files and so the error is not obvious. It would be nice if there was some more explicit testing around this code, if nothing else it would act as more documentation. Happy to spin into a new issue